pre_processing/orac_preproc.F90

```diff
@@ -486,6 +490,9 @@ subroutine orac_preproc(mytask, ntasks, lower_bound, upper_bound, driver_path_fi
    nargs = command_argument_count()
 #endif
 
+   ! Minimal output to notify start and end of programme
```

Comment:
Could this have `if (verbose)` at the front so we maintain silent operation by default?

Reply:
I put this in (and the finishing statement) precisely because completely silent operation can be problematic (especially if running in a batch environment), as it is sometimes not entirely obvious if the code actually ran!